README.md

```diff
@@ -1 +1,89 @@
-# first-capstone-conference
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+[![Twitter Badge](https://img.shields.io/badge/Twitter-Profile-informational?style=flat&logo=twitter&logoColor=white&color=1CA2F1)](https://twitter.com/ZikaZaki)
+[![LinkedIn Badge](https://img.shields.io/badge/LinkedIn-Profile-informational?style=flat&logo=linkedin&logoColor=white&color=0D76A8)](https://www.linkedin.com/in/zakariyaalkhamisisap)
+[![CodePen Badge](https://img.shields.io/badge/CodePen-Profile-informational?style=flat&logo=codepen&logoColor=white&color=black)](https://codepen.io/zikazaki)
+
+# ZikaZaki-Capstone-Project
+
+> This project is my first **capstone** project which is part of my **Microverse** exercises. This project follows the **Mobile First** approach, in order to make it prepared and responsive for different screen sizes. However, the goal here is to master and focus on best practices and professional **Simple, Visual, Thorough, & Looking-Good** shared traits.
+
+## Built With ⚙️
+
+- [x] Language: HTML, CSS, & JavaScript
+- [x] Linters: Lighthouse, Webhint, Stylelint and ESLint
+- [x] Code Editor: VS Code
+- [x] [Design Guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
+
+## Live Demo (Deployed on GitHub Pages)
+
+[![GitHub Pages](https://img.shields.io/badge/GitHub-Pages-blueviolet)](https://github.com/ZikaZaki/first-capstone-conference)
+
+[Live Demo Link](hhttps://github.com/ZikaZaki/first-capstone-conference)
```

Comment:
- [ ] You should deploy the project to GitHub pages. It is really easy, you can follow [this guide ](https://www.codecademy.com/article/f1-u3-github-pages)to deploy your project.